src/features/operator/Fee/FeeWithdraw/index.tsx

```diff
@@ -36,6 +50,33 @@ export const FeeWithdraw = (): JSX.Element => {
       ? 'N/A'
       : fromSatsToUnitOrFractional(feeBalance?.totalBalance, 8, true, lbtcUnit);
 
+  const feeFiatAmount = useMemo(() => {
```

Comment:
Use the helper function `convertAssetAmountToCurrency` to compute feeFiatAmount. Don't use `useMemo`